Makefile

```diff
@@ -8,12 +8,25 @@ doc:
 	@echo Generated documentation: "file://"$$(readlink -f doc/build/html/index.html)
 	@echo
 
+doc3:
+	python3 setup.py build_sphinx
+	@echo
+	@echo Generated documentation: "file://"$$(readlink -f doc/build/html/index.html)
+	@echo
+
```

Comment:
this part is not really needed. a separate `tox` environment for building the docs is perhaps a better solution.